script/pkgmacos

```diff
@@ -0,0 +1,108 @@
+#!/bin/zsh
+set -e
+
+print_help() {
+    cat <<EOF
+To build a universal pkg for macOS:
+  script/pkgmacos <tag-name>
+
+To build and sign set APPLE_DEVELOPER_INSTALLER_ID environment variable before.
+For example, if you have a signing identity with the identifier 
+"Developer ID Installer: Your Name (ABC123DEF)" set it in the variable.
+EOF
+}
+
+if [ $# -eq 0 ]; then
+  print_help >&2
+  exit 1
+fi
+
+tag_name=""
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+  -h | --help )
+    print_help
+    exit 0
+    ;;
+  -* )
+    printf "unrecognized flag: %s\n" "$1" >&2
+    exit 1
+    ;;
+  * )
+    tag_name="$1"
+    shift 1
+    ;;
+  esac
+done
```

Comment:
**Are there any conditions we should short circuit running this script?**

- Running it on unsupported OS
- If `pkgbuild` or `productbuild` aren't installed
- ...

Reply:
Not really required but it's better.
I adopted your suggested requirements and as unsupported OS i set all macOS versions prior to macOS 13 (Ventura).
For Linux it's not accounted for as it would exit with code 127 at the start anyways as `sw_vers` is not available on Linux.

Reply:
Do you think that's sufficient for this build script or should i also write a safe fail for Linux?